_posts/2022-11-17-what-makes-a-python-package-healthy.md

```diff
@@ -0,0 +1,536 @@
+---
+layout: single
+title: "What Makes a Python Open Source Package Healthy? A Conversation on Twitter."
+excerpt: "How should pyOpenSci measure Python open source package health and level of maintenance and usability? Here I summarize a conversation held on twitter around this very topic. Feedback is welcome!"
+author: "Leah Wasser"
+permalink: /blog/what-makes-open-source-python-package-healthy
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - highlight
+  - python-packaging
+  - peer-review
+toc: true
+comments: true
+---
+
+
+In previous posts I discussed:
+
+* [Why Python open source software matters to scientists (and package maintainers too)]()
+* [Why collecting metrics surrounding package health matters]()
+
+In this post, I'll summarize a conversation that was held on twitter that gaged 
+what the community thought about metrics to track the health of scientific 
+Python open source packages. 
+
+## Packages and open source software, a few terms to clarify
+
+* When I say Python package or Python open source software, i'm referring to a 
```

Comment:
```suggestion
* When I say Python package or Python open source software, I'm referring to a 
```

capitalized "I'm"

Reply:
:thinking: this sounds a bit repetitive, specifically the use of "open source" in both sentences. Maybe:

> * When I say Python package, I'm referring to software anyone can install, for example from PyPI or off of GitHub. 
> * When I say "open source" or "free and open source software" I'm referring to code that is free to download where the source code itself is openly available for anyone to see.